reflekt-plugin/src/main/kotlin/io/reflekt/plugin/utils/Util.kt

```diff
@@ -80,3 +83,19 @@ fun <T : Enum<T>> enumToRegexOptions(values: Array<T>, transform: T.() -> String
 
 fun <T : Enum<T>> String.toEnum(values: Array<T>, transform: T.() -> String): T =
     values.first { it.transform() == this }
+
+fun KotlinType.stringRepresentation(): String {
```

Comment:
why it's not in TypeStringRepresentation file?

Reply:
Since we don't have access to `KotlinType` in the TypeStringRepresentation file